awx/ui/client/src/network-ui/models.js

```diff
@@ -661,6 +661,29 @@ Group.prototype.update_membership = function (devices, groups) {
     return [old_devices, this.devices, device_ids, old_groups, this.groups, group_ids];
 };
 
+Group.prototype.is_in_breadcrumb = function(viewport){
+    var groupY1 = this.top_extent();
+    var groupX1 = this.left_extent();
+    var groupY2 = this.bottom_extent();
+    var groupX2 = this.right_extent();
+
+    var viewportY1 = viewport.top_extent();
+    var viewportX1 = viewport.left_extent();
+    var viewportY2 = viewport.bottom_extent();
+    var viewportX2 = viewport.right_extent();
+
+    if ( viewportX1 > groupX1 &&
```

Comment:
Is this indentation standard?

Reply:
eh no, I just did that for reasons that can't be explained. I can change that. @benthomasson